### ISSUE TITLE
Request for LED Strip

### ISSUE THREAD

**Issue by jabelone** [Templarian/MaterialDesign#5117]
I have:

- [*] [Searched all issues](https://github.com/Templarian/MaterialDesign/issues) to make sure there isn't a request for this icon.
- [*] [Searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't exist.
- [*] Only requested a single icon (or a few near-identical ones) in this issue. 

## Usage

In my home automation dashboard to represent the LED strip lighting that are under my cupboards.

## Examples

Basically it should look the same as the existing `led-strip` icon but it should be horizontal, and it shouldn’t have the little “legs” sticking out.

**Comment by Templarian** [comment]
Wouldn't `led-strip-variant` be the correct icon for that?

**Comment by jabelone** [comment]
I feel like it doesn't represent a horizontal LED strip very well. What I was thinking was something like this:
![led-strip](https://user-images.githubusercontent.com/9560876/80331553-b4995200-888b-11ea-954b-1fb57a937abd.png)

Or potentially something like this:
![led-strip](https://user-images.githubusercontent.com/9560876/80332224-90d70b80-888d-11ea-9ef1-736376be4335.png)

**Comment by jabelone** [comment]
Here's an image I found online for the type of LED strip lighting I'm talking about.

<img width="592" alt="Screen Shot 2020-04-27 at 1 52 37 pm" src="https://user-images.githubusercontent.com/9560876/80332512-5d48b100-888e-11ea-8375-a6ca9e14e0c8.png">

**Comment by Templarian** [comment]
I have Hue strips installed under my cabnets like that. Surprisingly bright. I think the `led-strip-variant` kind of works for any use of light strips, but maybe `led-down` / `led-up` light to represent light direction? Could work for these.

**Comment by jabelone** [comment]
I guess it's personal preference, but it would be great to have an icon that closely matches it instead of a generic `led-strip-variant`. I think an `led-down` / `led-up` is a great idea :)
